tests/stdlib/tosenv.nim

```diff
@@ -1,22 +1,41 @@
 discard """
   matrix: "--threads"
   joinable: false
-  target: "c cpp"
+  targets: "c cpp"
 """
+import os
 
-import os, system
+block delEnv:
+    const dummyEnvVar = "DUMMY_ENV_VAR" # This env var wouldn't be likely to exist to begin with
```

Comment:
2 spaces for indentation, not 4

Reply:
```suggestion
    const dummyEnvVar = "D20210720T144752"
```
(either use a guid-like string as this or prefix with test name eg: NIM_TOSENV_VAR)